src/components/confetti/index.js

```diff
@@ -12,11 +12,15 @@ const confettiStyles = () => ({
     justifyContent: 'center',
   },
   text: {
-    fontSize: '30px',
+    fontSize: '40px',
+    fontWeight: 300,
+  },
+  notifie: {
+    fontSize: '25px',
   },
 });
 
-const StyledConfetti = ({ classes }) => (
+const Confetti = ({ classes, notifie }) => (
```

Comment:
Why name the variable `notifie`?

Reply:
I didn't think of something better, any text user the success message should give the user information relevant to them at the end of the process.